helm_resource/helm-apply-helper.py

```diff
@@ -46,8 +47,27 @@
 subprocess.check_call(install_cmd, stdout=sys.stderr)
 
 print("Running cmd: %s" % get_cmd, file=sys.stderr)
-out = subprocess.check_output(get_cmd)
+out = subprocess.check_output(get_cmd).decode('utf-8')
+
+# We have to do namespace defaulting ourselves :(
+# See: https://github.com/tilt-dev/tilt-extensions/issues/374
+def add_default_namespace(yaml, namespace):
+  divider = '%s---%s' % (os.linesep, os.linesep)
```

Comment:
Should we skip if namespace is not provided?
```suggestion
  if not namespace:
    return yaml
  divider = '%s---%s' % (os.linesep, os.linesep)
```

Reply:
ooh good catch! fixed